Tasks/VsTest/task.json

```diff
@@ -38,30 +38,45 @@
       "helpMarkDown": "Filter criteria for the tests."
     },
     {
+      "name": "platform", 
+      "type": "string", 
+      "label": "Platform", 
+      "defaultValue":"", 
+      "required":false
+    },
+    {
+      "name": "configuration", 
+      "type": "string", 
+      "label": "Configuration", 
+      "defaultValue":"", 
+      "required":false 
+    },
+    {
       "name": "runSettingsFile",
-      "type": "filePath",
+      "type": "string",
```

Comment:
Why did you change this to a string?  This should be a file path so I optionally get a picker and the infrastructure will automatically try to convert a relative path to a full path.